src/lib-components/SectionStaffSubjectLibrarian.vue

```diff
@@ -0,0 +1,117 @@
+<script
+  setup
+  lang="ts"
+>
+import type { PropType } from 'vue'
+
+// TYPESCRIPT
+import type { BlockStaffListItemType } from '@/types/types'
+
+import BlockStaffSubjectLibrarian from '@/lib-components/BlockStaffSubjectLibrarian.vue'
+
+const { items, tableHeaders } = defineProps({
+  items: {
+    type: Array as PropType<BlockStaffListItemType[]>,
+    default: () => [],
+  },
+  tableHeaders: {
+    type: Array,
+    default: () => [],
+  },
+})
+</script>
+
+<template>
+  <table class="section-staff-subject-librarian">
+    <caption>
+      Subject Librarians
+    </caption>
+    <thead>
+      <tr>
+        <th
+          v-for="(header, index) in tableHeaders"
+          :key="index"
+        >
+          {{ header }}
+        </th>
+      </tr>
+    </thead>
+
+    <tbody>
+      <BlockStaffSubjectLibrarian
+        v-for="(item, index) in items"
+        :key="`${index}-${item.subjectArea}`"
+        :subject-area="item.subjectArea"
+        :staff-name="item.staffName"
+        :to="item.to"
+        :alternative-name="item.alternativeName"
+        :language="item.language"
+        :job-title="item.jobTitle"
+        :departments="item.departments"
+        :locations="item.locations"
+        :email="item.email"
+        :phone="item.phone"
+        :consultation="item.consultation"
+        class="subject-librarian-item"
+      />
```

Comment:
This component doesn't seem to be rendering correctly quite yet in the storybook, looks like fields are missing (trying to attach a screenshot showing the branch storybook vs the default)
<img width="956" alt="Screenshot 2024-04-04 at 12 34 11 PM" src="https://github.com/UCLALibrary/ucla-library-website-components/assets/7073987/87444c5b-b32a-45a2-bad8-081d06775382">
<img width="956" alt="Screenshot 2024-04-04 at 12 33 56 PM" src="https://github.com/UCLALibrary/ucla-library-website-components/assets/7073987/f2644b72-a865-4b60-b374-52c304a21349">